src/ripple/app/tests/Regression_test.cpp

```diff
@@ -157,11 +159,58 @@ struct Regression_test : public beast::unit_test::suite
         test256r1key (becky);
     }
 
+    static
+    std::unique_ptr<Config>
+    makeConfig(std::map<std::string, std::string> extra = {})
```

Comment:
I was planning to write some more tests that would reuse it, but since I haven't, and may not anytime soon (and they may belong in `TxQ_test.cpp` anyway when I do), a lambda will be fine.